tests/src/nested_scopes.rs

```diff
@@ -119,7 +119,7 @@ fn nested_scopes(device: &wgpu::Device, queue: &wgpu::Queue) {
                     ),
                 ],
             ),
-            expected_scope("e2_s0", Requires::Timestamps, []),
+            expected_scope("e2_s0", Requires::TimestampsInEncoders, []),
```

Comment:
as mentioned elsewhere, this test needs to be expanded / broken up to account for all 3 features instead of just 2